src/oop/user_defined_classes/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,28 @@
+# oop-user_defined_classes-medium-q2
+
+## Java
+
+Little boys love to play with cars and trucks and watch things move. 
+Make a Java class called Car that has attributes called colour, 
+horsePower, and yearMade. This class also has a constructor for 
+initializing the attributes and another method called printSpecs() 
+that displays the colour and horsepower of the vehicle.
+
+
+### Car Class
+
+The `Car` class you are given has the following items:
```

Comment:
For the medium, you can probably just give them the constructor, and then they can implement a basic method instead

Reply:
Specifically for this question. For another question, you can give them the attributes, and have them implement the constructor